ansible/tasks/logging/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: windows
```

Comment:
How do you plan to test this?

Reply:
Test plan for Ansible lives [here](https://docs.google.com/document/d/1tYvCHQ9toP-ukIDVqnjTvRpTVGLBGFeaXJtzLaw3jaM/edit)